scripts/es-integration-test.sh

```diff
@@ -87,21 +89,21 @@ bring_up_storage() {
   for retry in 1 2 3
   do
     echo "attempt $retry"
-    if [ ${distro} = "elasticsearch" ]; then
-      cid=$(setup_es ${version})
-    elif [ ${distro} == "opensearch" ]; then
-      cid=$(setup_opensearch ${version})
+    if [ "${distro}" = "elasticsearch" ]; then
+      cid=$(setup_es "${version}")
+    elif [ "${distro}" == "opensearch" ]; then
+      cid=$(setup_opensearch "${version}")
     else
       echo "Unknown distribution $distro. Valid options are opensearch or elasticsearch"
       usage
     fi
-    wait_for_storage ${distro} "http://localhost:9200" ${cid}
+    wait_for_storage "${distro}" "http://localhost:9200" "${cid}"
     if [ ${db_is_up} = "1" ]; then
       break
     fi
   done
   if [ ${db_is_up} = "1" ]; then
-    trap "teardown_storage ${cid}" EXIT
+    trap 'teardown_storage ${cid}' EXIT
```

Comment:
`cid` is local here, we cannot extrapolate it later.

Reply:
should i disable it?

Reply:
yes